packages/common-common/src/permissions.ts

```diff
@@ -67,6 +432,39 @@ export function computePermissions(
     permission &= ~BigInt(assignment.deny);
     permission |= BigInt(assignment.allow);
   }
+  return permission;
+}
+
+type RoleObject = {
+  permission: Permission;
+  allow: Permissions;
+  deny: Permissions;
+};
+
+export function aggregatePermissions(
```

Comment:
Can this also have a comment describing what it does? Not sure exactly. Also a return type.